pylint/checkers/misc.py

```diff
@@ -20,9 +20,9 @@
 
 # pylint: disable=W0511
 
-import re
+import tokenize
```

Comment:
@justinnhli I'm not sure how this got in here, those changes haven't landed in master yet.